### PR TITLE
Fix breadcrumb block service

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

Fixing the last test for Blocks.

### CODE REVIEW

**Comment by jordisala1991** [comment]
Not sure if general problem or just our builds… @VincentLanglet

File: composer.json [hunk 1]
```diff
@@ -20,6 +20,12 @@
         }
     ],
     "homepage": "https://docs.sonata-project.org/projects/SonataPageBundle",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/jordisala1991/SonataSeoBundle"
+        }
+    ],
```
Comment (jordisala1991) [inline composer.json]:
```suggestion
```

File: composer.json [hunk 2]
```diff
@@ -30,7 +36,7 @@
         "sonata-project/doctrine-extensions": "^1.8 || ^2.0",
         "sonata-project/doctrine-orm-admin-bundle": "^4.0",
         "sonata-project/form-extensions": "^1.4",
-        "sonata-project/seo-bundle": "^3.0",
+        "sonata-project/seo-bundle": "dev-hotfix/fix-base-breadcrumb-block-service as 3.0",
```
Comment (jordisala1991) [inline composer.json]:
```suggestion
        "sonata-project/seo-bundle": "3.x-dev as 3.0",
```

File: composer.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
         "sonata-project/doctrine-extensions": "^1.8 || ^2.0",
         "sonata-project/doctrine-orm-admin-bundle": "^4.0",
         "sonata-project/form-extensions": "^1.4",
-        "sonata-project/seo-bundle": "^3.0",
+        "sonata-project/seo-bundle": "3.x-dev as 3.0",
```
Comment (VincentLanglet) [inline composer.json]:
```suggestion
        "sonata-project/seo-bundle": "^3.3.0",
```

File: composer.json [hunk 4]
```diff
@@ -30,7 +30,7 @@
         "sonata-project/doctrine-extensions": "^1.8 || ^2.0",
         "sonata-project/doctrine-orm-admin-bundle": "^4.0",
         "sonata-project/form-extensions": "^1.4",
-        "sonata-project/seo-bundle": "^3.0",
+        "sonata-project/seo-bundle": "^3.3.0",
```
Comment (jordisala1991) [inline composer.json]:
```suggestion
        "sonata-project/seo-bundle": "^3.3",
```

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**